doc(install): add Longhorn CLI in quick installation section (#958)
longhorn/website: content/docs/1.7.0/deploy/install/_index.md

- Talos Linux clusters: See [Talos Linux Support](../../advanced-resources/os-distro-specific/talos-linux-support).
- Container-Optimized OS: See [Container-Optimized OS Support](../../advanced-resources/os-distro-specific/container-optimized-os-support).

### Using the Longhorn Command Line Tool

> The `longhornctl` tool is a CLI interface to Longhorn operations. For more information, see [Command Line Tool (longhornctl)](../../advanced-resources/longhornctl/).

Reviewer: Removed a redundant 'interface'.
```suggestion
The `longhornctl` tool is a CLI for Longhorn operations. For more information, see [Command Line Tool (longhornctl)](../../advanced-resources/longhornctl/).
```

Author: Modified. Thanks.

> The `longhornctl` tool is a CLI interface to Longhorn operations to help you gather enough information about the factors. For more information, please see [Command Line Tool (longhornctl)](../../advanced-resources/longhornctl/).

Reviewer 2: ```suggestion
The `longhornctl` tool is a CLI interface to Longhorn operations. For more information, please see [Command Line Tool (longhornctl)](../../advanced-resources/longhornctl/).
```

Author: Modified. Thanks.

>BTW, also need to update [v2 volume Quick Start ](https://longhorn.io/docs/1.7.0/v2-data-engine/quick-start/). Add --enable-spdk to the two sub-commands.

Updated. Thanks

> The `longhornctl` tool is a CLI interface to Longhorn operations. For more information, please see [Command Line Tool (longhornctl)](../../advanced-resources/longhornctl/).
> 
> To check the prerequisites and configurations, please download the command line tool `longhornctl` and utilize the `check` sub-command:

Reviewer 3: ```suggestion
The `longhornctl` tool is a CLI interface to Longhorn operations. For more information, see [Command Line Tool (longhornctl)](../../advanced-resources/longhornctl/).

To check the prerequisites and configurations, download the tool and run the `check` sub-command:
```

Author: Modified. Thanks.

> Download the `longhornctl` tool binary file and use the `check` sub-command to identifies potential issues before usage before installing Longhorn：

Reviewer 2: ```suggestion
To check the prerequisites and configurations, please download the command line tool `longhornctl` and utilize the `check` sub-command:
```

Author: Modified. Thanks.

```shell
# For AMD64 platform
curl -sSfL -o longhornctl https://github.com/longhorn/cli/releases/download/v{{< current-version >}}/longhornctl-linux-amd64
# For ARM platform
curl -sSfL -o longhornctl https://github.com/longhorn/cli/releases/download/v{{< current-version >}}/longhornctl-linux-arm64

chmod +x longhornctl
./longhornctl check preflight
```

> Example result:

Reviewer 3: ```suggestion
Example of result:
```

Author: Modified. Thanks.

```shell
INFO[2024-01-01T00:00:01Z] Initializing preflight checker
INFO[2024-01-01T00:00:01Z] Cleaning up preflight checker
INFO[2024-01-01T00:00:01Z] Running preflight checker
INFO[2024-01-01T00:00:02Z] Retrieved preflight checker result:
worker1:
  info:
  - Service iscsid is running
  - NFS4 is supported
  - Package nfs-common is installed
  - Package open-iscsi is installed
  warn:
  - multipathd.service is running. Please refer to https://longhorn.io/kb/troubleshooting-volume-with-multipath/ for more information.
worker2:
  info:
  - Service iscsid is running
  - NFS4 is supported
  - Package nfs-common is not installed
  - Package open-iscsi is installed
```

> You can use the `install` sub-command to perform preflight dependencies installation and setup before installing Longhorn.

Reviewer 3: ```suggestion
Use the `install` sub-command to install and set up the preflight dependencies before installing Longhorn.
```

Author: Modified. Thanks.
